app/requests/enrolment_authorisation_request.py

```diff
@@ -1,6 +1,6 @@
 from app.requests import ValidRequest
 
 
-class NewEnrolmentRequest(ValidRequest):
-    course_id: str
+class EnrolmentAuthorisationRequest(ValidRequest):
     student_id: str
+    course_id: str
```

Comment:
Are these two fields enough for enrolment authorisation? If there is any, can you share document/info with me? I'm trying to understand it better :)

Reply:
> Are these two fields enough for enrolment authorisation? If there is any, can you share document/info with me? I'm trying to understand it better :)

When I started this project enrolment authorisation entity was already there with these two fields. I thought may be it had been worked on. But yeah I think there should be employer_id and shared_secret too. So he may receive callbacks in employer-admin.